Makefile

```diff
@@ -64,9 +64,15 @@ environment::
 	@yarn install
 	@ln -sf ../node/bin/node ./node_modules/.bin/node
 
+@install-create-user::
+	@$(COMPOSE_EXEC_ROOT) mkdir -p /home/hostuser
+	@$(COMPOSE_EXEC_ROOT) useradd -u $(HOST_USER) hostuser
+	@$(COMPOSE_EXEC_ROOT) chown hostuser /home/hostuser
+
 install::
 	$(MAKE) -s up
 	@time $(MAKE) -s -j 3 @install-githooks @install-composer @install-yarn
+	$(MAKE) -si @install-create-user
```

Comment:
Couldn't this actually run in parellel with the others?

Reply:
Good point. I moved it to the make up command und run it in parallel with the chmod command for /data